llmware/library.py

```diff
@@ -724,7 +1137,18 @@ def add_pdf_by_ocr(self, input_folder=None):
 
     def add_pdf(self, input_folder=None):
 
-        """ Convenience method to directly add PDFs only - note, in most cases, 'add_files' is the better option."""
+        """ Convenience method to directly add PDFs only - note, in most cases, 'add_files' is the better option.
```

Comment:
Delete space before you start the doc.
```suggestion
        """Convenience method to directly add PDFs only - note, in most cases, 'add_files' is the better option.
```